modules/apps/map/map-google-maps/src/main/resources/META-INF/resources/js/GoogleMapsSearch.es.js

```diff
@@ -29,7 +28,7 @@ class GoogleMapsSearch extends State {
 	 */
 	constructor(...args) {
 		super(...args);
-		const inputNode = toElement(this.inputNode);
+		const inputNode = this.inputNode;
```

Comment:
Are we sure this is a node and not a selector?

Reply:
I guess so, based on https://github.com/liferay-frontend/liferay-portal/pull/480/files#diff-ae8e74ccc522db708780b72b0e4528a1dfaf87e7cd2a960c053a5c2aa6c67e91R112 👍